makexpi.sh

```diff
@@ -101,7 +100,26 @@ fi
 # Prepare packages suitable for uploading to EFF and AMO, respectively.
 [ -d pkg ] || mkdir pkg
 rsync -aL --delete --delete-excluded --exclude /chrome/content/rules src/ pkg/xpi-eff
-cp -a translations/* pkg/xpi-eff/chrome/locale/
+
+# START: The next lines are for embedded WebExtensions, and can be deleted after the full transition to WebExtensions
+[ -d pkg/xpi-eff/webextension ] || mkdir pkg/xpi-eff/webextension
+rsync -aL --delete chromium/ pkg/xpi-eff/webextension/
+
+mkdir pkg/xpi-eff/webextension/_locales/
+python2.7 utils/chromium-translations.py translations/ pkg/xpi-eff/webextension/_locales/
+python2.7 utils/chromium-translations.py src/chrome/locale/ pkg/xpi-eff/webextension/_locales/
+
+do_not_ship="*.py *.xml icon.jpg"
+rm -f pkg/xpi-eff/webextension/$do_not_ship
```

Comment:
This is wrong; it will delete *.xml and icon.jpg in the current directory. Instead just add one line per thing. Or possible skip this line? It seems like it's been broken, but hasn't had an ill effect.